packages/flutter/lib/src/material/tabs.dart

```diff
@@ -817,28 +823,40 @@ class _TabBarViewState extends PageableListState<TabBarView> implements TabBarSe
     _selection = null;
   }
 
-  void _updateItems(int first, int second, [int third]) {
+  void _updateItemsWithChildren(int first, int second, [int third]) {
```

Comment:
_updateItemsFromChildren? With makes it sound like you're updating both or something.